src/ServiceControl.Persistence.RavenDb5/RavenDBPersisterSettings.cs

```diff
@@ -27,4 +27,6 @@ class RavenDBPersisterSettings : PersistenceSettings
     public const int DatabaseMaintenancePortDefault = 33334;
     public const int ExpirationProcessTimerInSecondsDefault = 600;
     public const string LogsModeDefault = "Information";
+
+    public override bool MessageBodiesAlwaysStoredInFailedMessage => true;
```

Comment:
Why does this exist?

Reply:
Because the Raven3.5 persister might store it text-encoded in the document or it might store it in body storage, depending on reasons. When you're doing recoverability you have to figure out where it is, so this just tells it "Raven5 only does it one way".

If we remove the Raven3.5 code then we could simplify everything and remove this property as well.